src/pretalx/orga/forms/cfp.py

```diff
@@ -140,6 +145,7 @@ class Meta:
             "submission_public_review": "feature_flags",
             "present_multiple_times": "feature_flags",
             "mail_on_new_submission": "mail_settings",
+            "show_duration_in_submission_type": "feature_flags",
```

Comment:
Why did you add this flag here? You added it on the model, not as a feature flag, so why configure it as part of the feature flags JSON field?